services/app-api/emailer/emailer.ts

```diff
@@ -112,6 +121,18 @@ function newLocalEmailer(config: EmailConfiguration): Emailer {
             ${'(¯`·.¸¸.·´¯`·.¸¸.·´¯·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´¯`·.¸¸.·´)'}
         `)
         },
+        sendUnlockPackageCMSEmail: async (
+            unlockEmailData: UnlockEmailData
+        ) => {
+            const emailData = unlockPackageCMSEmail(unlockEmailData, config)
+            const emailRequestParams = getSESEmailParams(emailData)
```

Comment:
I think it would be nice to extract this bit out into its own function since we are repeating it 4 times in the localEmailer. getSeSRemailParams and the console.log could be in their own function